commons-db/src/main/java/com/palantir/nexus/db/pool/JdbcConfig.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.nexus.db.pool;
+
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import com.palantir.nexus.db.manager.DBConfig;
+
+public class JdbcConfig {
+
+    public static Properties getPropertiesFromDbConfig(DBConfig dbConfig) {
+        Properties properties = new Properties();
+
+        properties.setProperty("user", dbConfig.getDbLogin());
+        if (dbConfig.getDbDecryptedPassword() != null) {
+            properties.setProperty("password", dbConfig.getDbDecryptedPassword());
+        }
+
+        Integer socketTimeout = dbConfig.getSocketTimeoutSeconds();
+        Integer connectTimeout = dbConfig.getConnectTimeoutSeconds();
```

Comment:
nit: probably worth `s/Timeout/TimeoutInSeconds/` on these two to make it clear as Postgres and Oracle drivers handle their units differently